pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -263,9 +262,14 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 		}
 
 		if job.PodGroup.Spec.MinResources == nil {
+			klog.V(4).Infof("job %s MinResources is null.", job.Name)
 			return util.Permit
 		}
 		minReq := job.GetMinResources()
+
+		klog.V(4).Infof("job %s min resource<%v>", job.Name, minReq)
```

Comment:
These logs can be removed which are just for test

Reply:
not only for test，but also for debug when the plugin behavior doesn't meet expectations. And the log level is changed to 5